daprdocs/content/en/operations/performance-and-scalability/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+type: docs
+title: "Performance and Scalability"
+linkTitle: "Performance and Scalability"
+weight: 300
+description: "Benchmarks and guidelines"
+---
+Dapr allows developers to inject sidecars next to their apps that make it easy to write distribured applications. The Dapr APIs allow developers to manage state at scale, trigger their apps from external sources, fetch secrets, publish and consume messages, and call other apps in a secured manner with build in telemetry and resiliency.
+
+Dapr is designed to be highly performant and consume minimal overhead in terms of compute resources.
+
+## System overview
+
+Dapr consists of a data plane (the sidecar that runs next to your app) and a control plane that configures the sidecars and provides additional capabilities like cert management.
+
+### Self-hosted components
+
+* Sidecar (Data Plane)
+* Sentry (Optional, Control Plane)
+* Placement (Optional, Control Plane)
+
+### Kubernetes components
+
+* Sidecar (Data Plane)
+* Sentry (Optional, Control Plane)
+* Placement (Optional, Control Plane)
+* Operator (Control Plane)
+* Sidecar Injector (Control Plane)
+
+
+## Service Invocation Performance for Dapr v0.11.3
+
```

Comment:
These 3 paragraphs should be removed and a link added to https://docs.dapr.io/developing-applications/building-blocks/service-invocation/service-invocation-overview/.  You may want to read this article to give any feedback

Reply:
About this, I want to keep it. because of the sidecar nature of Dapr, some people will arrive at the performance section first and need to know what building block this refers to and what it is without checking the concepts section. I've added a link to  the service invocation overview.